crates/router/src/core/admin.rs

```diff
@@ -414,6 +414,8 @@ pub async fn create_payment_connector(
         business_country,
         business_label,
         business_sub_label: req.business_sub_label,
+        created_at: common_utils::date_time::now(),
+        modified_at: common_utils::date_time::now(),
```

Comment:
When the connector is being modified, the modified time also should be updated. Is this being done already?

Reply:
Yes.
This is how `MerchantConnectorAccountUpdateInternal` is defined. 
modified_at is still `Option<time::PrimitiveDateTime>` here. Should I change it to `time::PrimitiveDateTime`?
```
pub struct MerchantConnectorAccountUpdateInternal {
    merchant_id: Option<String>,
    connector_type: Option<storage_enums::ConnectorType>,
    connector_account_details: Option<Secret<serde_json::Value>>,
    test_mode: Option<bool>,
    disabled: Option<bool>,
    merchant_connector_id: Option<String>,
    payment_methods_enabled: Option<Vec<serde_json::Value>>,
    metadata: Option<pii::SecretSerdeValue>,
    frm_configs: Option<Secret<serde_json::Value>>,
    modified_at: Option<time::PrimitiveDateTime>,
}
impl From<MerchantConnectorAccountUpdate> for MerchantConnectorAccountUpdateInternal {
    fn from(merchant_connector_account_update: MerchantConnectorAccountUpdate) -> Self {
        match merchant_connector_account_update {
            MerchantConnectorAccountUpdate::Update {
                merchant_id,
                connector_type,
                connector_account_details,
                test_mode,
                disabled,
                merchant_connector_id,
                payment_methods_enabled,
                metadata,
                frm_configs,
            } => Self {
                merchant_id,
                connector_type,
                connector_account_details,
                test_mode,
                disabled,
                merchant_connector_id,
                payment_methods_enabled,
                metadata,
                frm_configs,
                modified_at: Some(common_utils::date_time::now()),
            },
        }
    }
}
```

Reply:
Yes, the modified field should be updated when an update is made. So this field should not be optional. But if somewhere the `default()` function is being used, then we need to remove the default if possible or implement default on `datetime` to provide the current time.